analysis/paper/paper.qmd

```diff
@@ -108,6 +108,7 @@ pdf_paths <- here(data_path, emu_theses$file_name)
 ```{r}
 #| label: extract-pdf-text
 #| echo: false
+#| cache: true
```

Comment:
This does not seem to work for me, but it would be great if we could cache the result of this code chunk to speed up the interactive development in the paper.

Reply:
I'll look into it (#60 ). I'm merging this PR too streamline the work on topic modelling